src/main/java/net/spy/memcached/MemcachedNodeROImpl.java

```
@@ -187,6 +187,11 @@ public int pendingOperationQueueSize() {
     return root.pendingOperationQueueSize();
   }
 
+  @Override
+  public boolean executeTlsHandshake() {
```

What is this class used for? Do we even need to provide an implementation here instead of throwing `UnsupportedOperationException();` like the other methods?

---

Seems like even if we don't need it it wouldn't hurt anything here anyways

---

This class is used for read-only clients. Every method needed for reading needs to provide an implementation, while every method that allows for writing should throw an `UnsupportedOperationException.